src/System.ValueTuple/tests/TupleTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if netcoreapp11
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Xunit;
+using System.Runtime.CompilerServices;
+
+namespace System.Tests
+{
+    public class TupleTests
+    {
+        [Fact]
+        public void Test()
+        {
+            // Subset of the Tuple tests related to ITuple
+            // from $/DevDiv/FX/Feature/NetFXDev1/QA/CLR/testsrc/CoreMangLib/System/Tuple/TupleTest.cs
```

Comment:
I'll remove.
Do you know why System.Tuple (implemented in System.Private.CoreLib) does not appear to have tests in corefx?